lib/plugins/expressionParser.js

```diff
@@ -163,6 +165,16 @@ function expressionApplier(context, typeInformation) {
 
         newAttribute.value = applyExpression(attribute.expression, context, typeInformation);
 
+        if (attribute.type === 'Integer') {
```

Comment:
I understand this piece of code is due to different from NGSIv1, attribute values in NGSIv2 could use other native types in JSON appart from string. I see the numeric an boolean types covered, but, what about `null` (which is valid value for attribute in NGSIv2?

I'm not fully sury about it, maybe the comment doesn't apply... could you elaborate on the need of this piece of code, please?

Reply:
Fixed in 0b20a6e218960a8a2732a3dbf78d7c4ac76b52a5